egs/wsj/s5/utils/validate_dict_dir.pl

```diff
@@ -186,7 +186,7 @@ sub check_lexicon {
     if (!defined $word) {
       print "--> ERROR: empty lexicon line in $lex\n"; set_to_fail();
     }
-    if ($word eq "<s>" || $word eq "</s>" || $word eq "<eps>") {
+    if ($word eq "<s>" || $word eq "</s>" || $word eq "<eps>" || $word eq "#0") {
```

Comment:
can you please make sure the mini-librispeech setup, at least till the first decoding, runs with these changes?